<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="WptIND-F7T37V3" failures="2" tests="5" name="Functional" time="184.991" errors="0" timestamp="2025-06-17T19:41:03 IST">
  <testcase classname="TestCase.TC_002_Verify_Login" name="verifyLog" time="31.056"/>
  <testcase classname="TestCase.TC_002_Verify_Login" name="verifyLog" time="30.866"/>
  <testcase classname="TestCase.TC_002_Verify_Login" name="verifyLog" time="31.175"/>
  <testcase classname="TestCase.TC_002_Verify_Login" name="verifyLog" time="30.976">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestCase.TC_002_Verify_Login.verifyLog(TC_002_Verify_Login.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verifyLog -->
  <testcase classname="TestCase.TC_003_ChangePassword" name="changepass" time="45.521">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestCase.TC_003_ChangePassword.changepass(TC_003_ChangePassword.java:90)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- changepass -->
  <testcase classname="TestCase.TC_001_LoginVerify" name="verify" time="0.0">
    <ignored/>
  </testcase> <!-- verify -->
</testsuite> <!-- Functional -->
